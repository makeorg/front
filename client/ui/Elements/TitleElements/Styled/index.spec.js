import React from 'react';
import renderer from 'react-test-renderer';
import {
  SecondLevelTitleStyle,
  ThirdLevelTtitleStyle,
  FourthLevelTitleStyle,
} from '.';

describe('TitleElements', () => {
  describe('SecondLevelTitleStyle', () => {
    it('snapshot by default', () => {
      const component = renderer
        .create(<SecondLevelTitleStyle>H2</SecondLevelTitleStyle>)
        .toJSON();
      expect(component).toMatchSnapshot();
    });
  });
  describe('ThirdLevelTtitleStyle', () => {
    it('snapshot by default', () => {
      const component = renderer
        .create(<ThirdLevelTtitleStyle>H3</ThirdLevelTtitleStyle>)
        .toJSON();
      expect(component).toMatchSnapshot();
    });
  });
  describe('FourthLevelTitleStyle', () => {
    it('snapshot by default', () => {
      const component = renderer
        .create(<FourthLevelTitleStyle>H4</FourthLevelTitleStyle>)
        .toJSON();
      expect(component).toMatchSnapshot();
    });
  });
});
