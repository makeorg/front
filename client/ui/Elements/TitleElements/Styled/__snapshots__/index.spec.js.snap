// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TitleElements FourthLevelTitleStyle snapshot by default 1`] = `
.c0 {
  font-size: 0.75rem;
}

@media (min-width:47.94rem) {
  .c0 {
    font-size: 1rem;
  }
}

<h4
  className="c0"
>
  H4
</h4>
`;

exports[`TitleElements SecondLevelTitleStyle snapshot by default 1`] = `
.c0 {
  font-size: 1rem;
}

@media (min-width:47.94rem) {
  .c0 {
    font-size: 1.375rem;
  }
}

<h2
  className="c0"
>
  H2
</h2>
`;

exports[`TitleElements ThirdLevelTtitleStyle snapshot by default 1`] = `
.c0 {
  font-size: 0.875rem;
}

@media (min-width:47.94rem) {
  .c0 {
    font-size: 1.125rem;
  }
}

<h3
  className="c0"
>
  H3
</h3>
`;
